src/content/docs/en/developers/guides/running-a-scroll-node.mdx

```diff
@@ -139,6 +139,161 @@ Running the node in Docker might have a significant impact on node performance.
 - Check logs: `docker logs --tail 10 -f l2geth-docker`
 - Stop the container: `docker stop l2geth-docker`
 
+---
+## Run L2geth in L1 follower mode
+
+<Aside type="tip">
+L1 follower mode runs a node that only follows the finalized L1 and DA to read and derive the L2 chain. In this mode the node does not participate directly in the L2.
+This mode is useful for reconstructing the L2 state by only following the L1.
+</Aside>
+
+Run `l2geth` with the `--da.sync` flag. Provide blob APIs and beacon node with
+- `--da.blob.beaconnode "<L1 beacon node>"` (recommended, if beacon node supports historical blobs)
+- `--da.blob.blobscan "https://api.blobscan.com/blobs/"` `--da.blob.blocknative "https://api.ethernow.xyz/v1/blob/"` for mainnet
+- `--da.blob.blobscan "https://api.sepolia.blobscan.com/blobs/"` for Sepolia.
+
+Strictly speaking only one of the blob providers is necessary, but during testing blobscan and blocknative were not fully reliable. That's why using a beacon node with historical blob data is recommended (can be additionally to blobscan and blobnative).
+
+### mainnet
+```bash
+./build/bin/geth --scroll \
+--datadir "tmp/mainnet-l2geth-datadir" \
+--gcmode archive \
+--http --http.addr "0.0.0.0" --http.port 8545 --http.api "eth,net,web3,debug,scroll" \
+--da.sync=true \
+--da.blob.blobscan "https://api.blobscan.com/blobs/" --da.blob.blocknative "https://api.ethernow.xyz/v1/blob/" \
+--da.blob.beaconnode "<L1 beacon node>" \
+--l1.endpoint "<L1 RPC node>" \
+--verbosity 3
+```
+
+A full sync will take about 2 weeks depending on the speed of the RPC node, beacon node and the local machine. Progess is reported as follows for every 1000 blocks applied:
+
+```bash
+INFO [08-01|16:44:42.173] L1 sync progress                         blockhain height=87000 block hash=608eec..880ebd root=218215..9a58a2
+```
+
+### Sepolia
+```bash
+./build/bin/geth --scroll-sepolia \
+--datadir "tmp/sepolia-l2geth-datadir" \
+--gcmode archive \
+--http --http.addr "0.0.0.0" --http.port 8545 --http.api "eth,net,web3,debug,scroll" \
+--da.sync=true \
+--da.blob.blobscan "https://api.sepolia.blobscan.com/blobs/" \
+--da.blob.beaconnode "<L1 beacon node>" \
+--l1.endpoint "<L1 RPC node>" \
+--verbosity 3
+```
+
+A full sync will take about 2-3 days depending on the speed of the RPC node, beacon node and the local machine. Progess is reported as follows for every 1000 blocks applied:
+
+```bash
+INFO [08-01|16:44:42.173] L1 sync progress                         blockhain height=87000 block hash=608eec..880ebd root=218215..9a58a2
+```
+
+
+### Troubleshooting
+You should see something like this shortly after starting:
```

Comment:
Maybe move this statement to right before the code block

Reply:
done